spec/sailor.spec.js

```diff
@@ -176,20 +142,33 @@ describe('Sailor', function () {
                     expect(fakeAMQPConnection.ack.callCount).toEqual(1);
                     expect(fakeAMQPConnection.ack.calls[0].args[0]).toEqual(message);
                 })
-                .done(done, done.fail);
+                .done(done, done); //todo: use done.fail after migration to Jasmine 2.x
         });
 
         it('should send request to API server to update keys', function (done) {
-
+            settings.FUNCTION = 'keys_trigger';
             var sailor = new Sailor(settings);
 
-            spyOn(sailor, "getStepInfo").andReturn({
-                function: "keys_trigger"
-            });
-
-            var nockScope = nock('http://apihost.com:80')
-                .put('/v1/accounts/1234567890', {keys: {oauth: {access_token: 'newAccessToken'}}})
-                .reply(200, "Success");
+            sailor.apiClient = {
```

Comment:
Wait, but I do expectations when I do mock for API client:
```
    retrieveStep: function(taskId, stepId) {
                       expect(taskId).toEqual('5559edd38968ec0736000003');
                       expect(stepId).toEqual('step_1');
```

So if I do refactoring, I still expect apiClient to have been called with certain arguments. Why can I fuckup a request body?